android/src/main/java/com/ticketmasterignite/retail/PrePurchaseFavoritesListener.kt

```diff
@@ -0,0 +1,27 @@
+package com.ticketmasterignite.retail
+
+import com.ticketmaster.discoveryapi.models.DiscoveryAbstractEntity
+import com.ticketmaster.discoveryapi.models.MemberInfo
+import com.ticketmaster.prepurchase.listener.TMPrePurchaseFavoritesListener
+import com.ticketmasterignite.GlobalEventEmitter
+
+class PrePurchaseFavoritesListener: TMPrePurchaseFavoritesListener {
+  override fun isAbstractEntityInFavorites(event: DiscoveryAbstractEntity): Boolean {
+    TODO("Not yet implemented")
+//    GlobalEventEmitter.sendEvent("igniteAnalytics", "prePurchaseSdkIsEventInFavorites")
```

Comment:
as above - are we planning to implement the commented out code?

Reply:
For now, no request internally or from integrators implement these, that method `TODO("Not yet implemented")` gave an error when left empty and some methods that expected a string to be returned gave an error so `return ""` was used. The rest that didn't complain have been left blank. 

As there is quite a lot, this can be ongoing work to add more on demand.